src/Data/Vec/Properties.agda

```diff
@@ -621,3 +739,6 @@ module _ {a} {A : Set a} where
   toList∘fromList : (xs : List A) → toList (fromList xs) ≡ xs
   toList∘fromList List.[]       = refl
   toList∘fromList (x List.∷ xs) = P.cong (x List.∷_) (toList∘fromList xs)
+
+-- -}
+-- -}
```

Comment:
This was probably not intended.

Reply:
Indeed.  It is owed to having to close a comment, even if it is supposed to last until the end of the file.  I thought there was a rejected proposal to relax it, but I cannot find it on the Agda bug tracker.  It would be nice if the lack of a closing -} would be a warning rather than an error.